### DISCUSSION TITLE
The third layer of the three-layer mosaic presents a different color

### DISCUSSION THREAD

**Discussion by Mikewangkang** [neperfepx/neper#743] [Q&A] [answered]
Hello everyone
I used neper to do a three-layer mosaic, but my third layer is the same color after visualization. In this way, my disanceng mosaic cannot be seen after dividing the grid. Is there any way to change the color of the third layer to be different? Below is my code and picture, can you guys help me?
best wishes
wangkang
![1693483000178](https://github.com/neperfepx/neper/assets/139696590/10ed523b-c65e-4b3f-ae9a-dbf181de51e2)
![1693483068920](https://github.com/neperfepx/neper/assets/139696590/34477673-2f35-4320-9b5d-15f301e689ad)
[morpho_msfile3.txt](https://github.com/neperfepx/neper/files/12485817/morpho_msfile3.txt)
[n_msfile2.txt](https://github.com/neperfepx/neper/files/12485818/n_msfile2.txt)
[n_msfile3.txt](https://github.com/neperfepx/neper/files/12485819/n_msfile3.txt)

**Comment by rquey** [comment, marked as answer]
You are probably using `-datacellcol "scaleid(1)"`. Using `-datacellcol "scaleid(2)"` or `-datacellcol id` should solve the problem.
